languages/go/reference/exercise-concepts/parallel-letter-frequency.md

```diff
@@ -0,0 +1,29 @@
+# Concepts of Parallel Letter Frequency
+
+## Concurrency
+
+The goal of this exercise is to learn to write concurrent code to count the letters inside different texts.
+
+## Goroutines
+
+In order to count concurrent the apppearance of different letters in separate texts it is important to be able to create goroutines with the `go` keyword, one for each text where the letters are counted.
+
+## Channel
```

Comment:
There is basically no good alternative to a channel here. It's common for students to use a mutex to solve the data race problem, but I'll explain to them that this also eliminates the concurrency!